src/commonMain/kotlin/com/hoc081098/flowext/cast.kt

```diff
@@ -53,3 +53,9 @@ public inline fun <reified T : Any> Flow<T?>.castNotNull(): Flow<T> = map { it!!
  */
 @Suppress("NOTHING_TO_INLINE")
 public inline fun <T> Flow<T>.castNullable(): Flow<T?> = this
+
+/**
+ * Adapt this `Flow<*>` to be a `Flow<R?>`.
```

Comment:
Add `At the collection time, if this [Flow] has any value that is not an instance of R, null will be emitted`